tests/countries/test_dominica.py

```diff
@@ -0,0 +1,378 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/python-holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.dominica import Dominica, DM, DMA
+from tests.common import CommonCountryTests
+
+
+class TestDominica(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(Dominica)
+
+    def test_country_aliases(self):
+        self.assertAliases(Dominica, DM, DMA)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(Dominica(years=1989))
+
+    def test_special_holidays(self):
+        self.assertHoliday(
+            "2009-07-28",
+            "2009-09-03",
+            "2010-01-04",
+            "2019-09-19",
+        )
+
+    def test_labour_day(self):
+        name = "Labour Day"
+
+        # May, 1st.
+        self.assertNoHoliday("1989-05-01")
```

Comment:
I think this check is redundant a bit, because we have already checked that there are no holidays at all in 1989.

Reply:
Fixed.